dirty.rb

```diff
@@ -0,0 +1,34 @@
+a =     'b'
+eval '1+1'
+"#{1}"
+begin
+  foo
+rescue
+end
+begin
+begin
+begin
+begin
+  fooaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
+rescue
+end
+  foo
+rescue
+end
+  foo
+rescue
+end
+  foo
+rescue
+end
+a =     'b'
+a =     'b'
+a =     'b'
+a =     'b'
+a =     'b'
+a =     'b'
+a =     'b'
+a =     'b'
+a =     'b'
```

Comment:
__[development][rubocop]__
* Useless assignment to variable - `a`. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)